src/GuzzleHttp/Command/Guzzle/GuzzleClient.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
Rather than using this client class, I would actually use an event subscriber, and inject it using the DI extension.